contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -17,13 +17,54 @@ def wait_for_main_window(application: Application):
     wait_for_condition(lambda: application.top_window().class_name() == "OrbitMainWindow", 30)
 
 
+def wait_for_connection_window(application: Application):
```

Comment:
Nit. We're already not following this above, but: Common practice to indicate that a method is "module private" is prefixing it with an underscore.

Reply:
Done (also for the one above.